spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -963,9 +960,11 @@ class DistriOptimizer[T: ClassTag] (
 
     prepareInput()
 
-    models = DistriOptimizer.initThreadModels(model, distDataset, criterion, state,
+    val modelsAndBroadcast = DistriOptimizer.initThreadModels(model, distDataset, criterion, state,
```

Comment:
(models, modelBroadcast) = ... ?

Reply:
Scala doesn't support this style. We only can init two variables like this, such as `val (a, b) = (1, 2)`. But can't assign to two `var`s, such as `var c:Int = 0; var d: Int = 0; (c, d) = (1, 2)`.